src/molecules/Helper/calculateAmount.js

```diff
@@ -0,0 +1,17 @@
+import ProductData from "../../assets/Data/ProductData";
+
```

Comment:
Only the component file starts with a capital letter
everything else should be in camel case